tool/mustachio/codegen_runtime_renderer.dart

```diff
@@ -191,7 +220,8 @@ class ${renderer._rendererClassName}${renderer._typeParametersString}
 
     if (property.isPrivate || property.isStatic || property.isSetter) return;
     _buffer.writeln("'${property.name}': Property(");
-    _buffer.writeln('getValue: ($typeName c) => c.${property.name},');
+    _buffer
+        .writeln('getValue: (Object c) => (c as $typeName).${property.name},');
```

Comment:
I don't object, but I am curious what the motivation is for this change?

Reply:
The code as it was was a runtime error. The `Map<String, Property>` must contain `Property<Object>` objects, and the functions of a `Property<Object>` object must take an `Object` as a parameter. A bummer, and potential performance hit, but necessary without some ingenuity. :(